server/views/pages/goal/review/index.njk

```diff
@@ -0,0 +1,61 @@
+{% extends "../../../partials/layout.njk" %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+{% set pageId = "review" %}
+{% set pageTitle = applicationName + " - Home" %}
+{% set mainClasses = "app-container govuk-body" %}
+
+{% block beforeContent %}
+  {% include "../../../partials/breadCrumb.njk" %}
+{% endblock %}
+
+{% block content %}
+
+  {% if errors.length > 0 %}
```

Comment:
I think you can probably remove this if block, and the `error-summary` import, because there will never be any errors on this screen? (because the `ReviewView` object does not contain an `errors` property)